### PR TITLE
feat: add logto email

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add logto email for demo.

Will optimize the access token cache in the future (LOG-5693)

## Testing

Local tested and UTs

## Checklist

- [x] `.changset-staged`
- [x] unit tests
- [ ] integration tests
- [ ] docs

OR

- [ ] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/connector-logto-email/logo.svg
Comment (darcyYe) [inline packages/connector-logto-email/logo.svg:1]: use GitHub logo as placeholder?

File: packages/connector-logto-email/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { defaultMetadata, defaultTimeout, emailEndpoint } from './constant.js';
+import { grantAccessToken } from './grant-access-token.js';
+import type { LogtoEmailConfig } from './types.js';
+import { logtoEmailConfigGuard } from './types.js';
+
+const sendMessage =
+  (getConfig: GetConnectorConfig): SendMessageFunction =>
+  async (data, inputConfig) => {
+    const config = inputConfig ?? (await getConfig(defaultMetadata.id));
+    validateConfig<LogtoEmailConfig>(config, logtoEmailConfigGuard);
```
Comment (darcyYe) [inline packages/connector-logto-email/src/index.ts:19]: will the config be seeded in the db directly without user's configuration?

**Review by darcyYe — APPROVED** [review]

File: packages/connector-logto-email/src/index.ts (hunk 1 above)
Comment (wangsijie) [reply packages/connector-logto-email/src/index.ts:19]: YES

File: packages/connector-logto-email/logo.svg
Comment (wangsijie) [reply packages/connector-logto-email/logo.svg:1]: will replace with new Logo ASAP

Comment (wangsijie) [reply packages/connector-logto-email/logo.svg:1]: updated

File: packages/connector-logto-email/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Logto email connector
```
Comment (gao-sun) [inline packages/connector-logto-email/README.md:1]: double check if we have code logic in core to remove logto connectors from normal installs (like mock connectors)

**Review by gao-sun — APPROVED** [review]

File: packages/connector-logto-email/README.md (hunk 2 above)
Comment (wangsijie) [reply packages/connector-logto-email/README.md:1]: Yes, see https://github.com/logto-io/logto/pull/3414

**Pull request merged**